dbt/models/default/default.vw_pin_status.sql

```diff
@@ -118,7 +141,8 @@ SELECT
     ddat.cdu_description AS dweldat_cdu_description,
     pdat.note2 AS pardat_note,
     pdat.class = '999' AS is_filler_class,
-    pdat.parid LIKE '%999%' AS is_filler_pin
+    pdat.parid LIKE '%999%' AS is_filler_pin,
+    permit.num_assessable_permits > 0 AS has_recent_assessable_permit
```

Comment:
I wonder if we want to preserver the number of assessable permits? I decided to lean in the direction of YAGNI. I'm already not totally convinced that this PIN status will be super useful outside the modeling code, but it will be easy to strip out in the future if that ends up being true.